doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -204,6 +204,8 @@ nRF Desktop
   * The :ref:`nrf_desktop_bootloader` and :ref:`nrf_desktop_bootloader_background_dfu` sections in the nRF Desktop documentation to explicitly mention the supported DFU configurations.
   * The documentation describing the :ref:`nrf_desktop_memory_layout` configuration to simplify the process of getting started with the application.
   * Changed the term *flash memory* to *non-volatile memory* for generalization purposes.
+  * The :ref:`nrf_desktop_usb_state` to use :c:func:`usb_hid_set_proto_code` function to set the HID Boot Interface protocol code.
+    The ``CONFIG_USB_HID_BOOT_PROTOCOL`` Kconfig option was removed and dedicated API needs to be used instead.
```

Comment:
This option is still available and you are using it in the code. 
Did you mean CONFIG_USB_HID_PROTOCOL_CODE?
Also, at the current state of the NCS it is deprecated, not removed, but I guess it will change after the upmerge.

Reply:
Nice catch. Updated :)